basic-spring-boot/inventory/hosts

```diff
@@ -0,0 +1,2 @@
+[seed-hosts]
+localhost
```

Comment:
please add `ansible_connection=local` here and remove from the README instruction

a la https://github.com/rht-labs/labs-ci-cd/blob/master/inventory/hosts#L2